### PR TITLE
Fix some links and typos

### PR DESCRIPTION
**Pull request by segovoni**
Links to:

- `sp_execute_remote` system stored procedure
- Azure Active Directory

Typos:

- Round brackets
- Spaces

### CODE REVIEW

**Review by scoriani — APPROVED** [review]

**Comment by segovoni** [comment]
For the absolute link: In the sql-docs repository I did not find the .md file relating to the new name of Azure Active Directory.

File: azure-sql/database/elastic-query-getting-started-vertical.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ INSERT INTO [dbo].[CustomerInformation] ([CustomerID], [CustomerName], [Company]
 
     The "master_key_password" is a strong password of your choosing used to encrypt the connection credentials. 
     The "username" and "password" should be the username and password used to log in into the Customers database (create a new user in Customers database if one does not already exists).
-    Authentication using Microsoft Entra ID ([formerly Azure Active Directory](/azure/active-directory/fundamentals/new-name)) with elastic queries is not currently supported.
+    Authentication using Microsoft Entra ID ([formerly Azure Active Directory](https://learn.microsoft.com/azure/active-directory/fundamentals/new-name)) with elastic queries is not currently supported.
```
Comment (rwestMSFT) [inline azure-sql/database/elastic-query-getting-started-vertical.md]:
```suggestion
    Authentication using Microsoft Entra ID ([formerly Azure Active Directory](/entra/fundamentals/new-name)) with elastic queries is not currently supported.
```

File: azure-sql/database/elastic-query-getting-started-vertical.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ For pricing information, see [SQL Database Pricing](https://azure.microsoft.com/
...
 * For a horizontal partitioning (sharding) tutorial, see [Getting started with elastic query for horizontal partitioning (sharding)](elastic-query-getting-started.md).
-* For syntax and sample queries for horizontally partitioned data, see [Querying horizontally partitioned data)](elastic-query-horizontal-partitioning.md)
-* See [sp\_execute \_remote](/sql/relational-databases/system-stored-procedures/sp-execute-remote-azure-sql-database) for a stored procedure that executes a Transact-SQL statement on a single remote Azure SQL Database or set of databases serving as shards in a horizontal partitioning scheme.
+* For syntax and sample queries for horizontally partitioned data, see [Querying horizontally partitioned data](elastic-query-horizontal-partitioning.md).
+* See [sp\_execute\_remote](/docs/relational-databases/system-stored-procedures/sp-execute-remote-azure-sql-database.md) for a stored procedure that executes a Transact-SQL statement on a single remote Azure SQL Database or set of databases serving as shards in a horizontal partitioning scheme.
```
Comment (rwestMSFT) [inline azure-sql/database/elastic-query-getting-started-vertical.md]:
```suggestion
* See [sp\_execute\_remote](/sql/relational-databases/system-stored-procedures/sp-execute-remote-azure-sql-database) for a stored procedure that executes a Transact-SQL statement on a single remote Azure SQL Database or set of databases serving as shards in a horizontal partitioning scheme.
```

**Comment by rwestMSFT** [comment]
@segovoni Owing to how our internal relative links work, I've rolled back two of your changes. Thanks for the other fixes.

**Comment by segovoni** [comment]
@rwestMSFT I saw the changes, thanks, that's good to know.

**Pull request merged**